WeRockIT-ui/src/App.vue

```diff
@@ -1,12 +1,9 @@
 <template>
   <div id="app">
     <div class="container mx-auto">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link> |
-      <router-link to="/speakers">Speakers</router-link> |
-      <router-link to="/wip">Schedule</router-link> |
-      <router-link to="/wip">Sponsors</router-link> |
-      <router-link to="/venue">Venue/Travel</router-link> |
+      <router-link to="/">Home</router-link> | <router-link to="/about">About</router-link> |
+      <router-link to="/speakers">Speakers</router-link> | <router-link to="/wip">Schedule</router-link> |
+      <router-link to="/wip">Sponsors</router-link> | <router-link to="/venue">Venue/Travel</router-link> |
```

Comment:
Why move these to being more condensed?
I felt they were pretty readable as one per line calls

Reply:
This was one of the circumstances that the configuration with the `yarn lint:autofix` kept changing it to this setup versus the way it was originally.  I think we will need to play around with the configuration in order to prevent this from happening. I agree that the original way was much better.

Reply:
I fixed it to not use the 'autosave' in VSCode and we just can use the 'yarn lint:autofix' and it doesn't fix it with the /* eslint-disable prettier/prettier */.  I think I need to work with my local setup more in order to get things working.

Reply:
Womp, even with the disable it is still auto fixing it.  Going to keep digging into this much more.